### ISSUE TITLE
Please remove any reference to a Concur Sandbox in our API documentation

### ISSUE THREAD

**Issue by joehottingerConcur** [concur/developer.concur.com#1335]
**Note: for API troubleshooting and support questions, please use the [Concur Developer Center Forum](https://forum.developer.concur.com/).**

## 

* **I am creating an issue for...**
  - [ ] New API reference pages
  - [X] Formatting/text changes
  - [ ] A bug report
  - [ ] Feature request

* **What organization or team are you associated with?**
[Concur Web Services]

* **What's the description of your issue?**

[We have been informed by Concur OPs that we don't provide Sandbox environments to our Standard Clients anymore.  Please remove or reword the following page from our API documentation: https://developer.concur.com/manage-apps/register.html.  This page creates confusion for our Standard Web Services clients because they interpret it to mean they get a Sandbox environment for API testing.  Help us serve our WS clients by explaining how we allow API testing within their entity by creating test users, test payment batches, test connected lists, etc.
]

* **If this issue is a bug...**
  * **What are the steps to reproduce the bug?:** [NA]
  * **Expected behavior:** [What you expected to happen]
  * **Actual behavior:** [What actually happened]

**Comment by bhague1281** [comment]
There is sandbox information throughout many of the documentation pages, especially the API guides. Could you provide some examples on what to replace this wording with? Specifically in regards to how to set up their environment to create test resources (if any setup is necessary).

Once that's done, we can take care of eliminating the sandbox creation process across the entire site.

**Comment by charlieconcur** [comment]
I think this issue asks us to make it clear a 'Sandbox' is only available for Professional SKUs and only in one place: `developer.concur.com/src/manage-apps/register.html`

I think that's a pretty easy, targeted change.

Brian, can you advise why this particular page is in the source as an `.html` file rather than a `.markdown` file?

**Comment by charlieconcur** [comment]
@dstaab I'd like for you to review this resource for accuracy as well.

https://github.com/concur/developer.concur.com/blob/preview/src/manage-apps/register.html

**Comment by dstaab** [comment]
Charlie - are you asking me to edit this page:
https://github.com/concur/developer.concur.com/blob/preview/src/manage-apps/register.html

**Comment by charlieconcur** [comment]
Yes, taking into consideration the feedback from @joehottingerConcur above.

**Comment by dstaab** [comment]
I think Joe needs to schedule a call with us to describe what his intentions are because the Partner-side of the business needs to use sandboxes.  @joehottingerConcur - please setup a conference call with Charlie and I.  thanks - Doug
